### PR TITLE
Feature/tcvp 1867

### PR DESCRIPTION
**Pull request by lfdgov**
# Description

This PR includes the following proposed change(s):

- feature/TCVP-1867

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- reviewed JJ workbench to verify written reasons functionality not broken
- reviewed DCF, specifically count details for adherence to wireframe, data saves and retrieves correctly

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/staff-portal/debug.log [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[1110/123021.464:ERROR:registration_protocol_win.cc(107)] CreateFile: The system cannot find the file specified. (0x2)
```
Comment (pbolduc) [inline src/frontend/staff-portal/debug.log:1]: please remove the debug log from commit

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts [hunk 2]
```diff
@@ -83,11 +126,63 @@ export class JJCountComponent implements OnInit {
         this.jjDisputedCount.lesserOrGreaterAmount = this.form.get('lesserOrGreaterAmount').value;
         this.jjDisputedCount.totalFineAmount = this.form.get('totalFineAmount').value;
         this.jjDisputedCount.includesSurcharge = this.inclSurcharge == "yes" ? this.IncludesSurcharge.Y : this.IncludesSurcharge.N;
+        if (this.jjDisputeInfo.hearingType === this.HearingType.CourtAppearance) {
+          this.jjDisputedCount.jjDisputedCountRoP.finding = this.form.get('jjDisputedCountRoP').get('finding').value;
```
Comment (pbolduc) [inline src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts:130]: can you remove the duplicated code. Extract `this.form.get('jjDisputedCountRoP')` to a variable at the top. Will make more readable and increase minification effectiveness.

File: src/frontend/staff-portal/src/app/services/lookups.service.ts [hunk 3]
```diff
@@ -69,7 +69,7 @@ export class LookupsService implements ILookupsService {
   }
 
   public get__statuteString(statute: Statute): string {
-    return statute.code + " " + statute.sectionText + " " + statute.descriptionText;
+    return statute.code + " " + statute.descriptionText;
```
Comment (pbolduc) [inline src/frontend/staff-portal/src/app/services/lookups.service.ts:72]: can use string interpolation.

**Review by pbolduc — CHANGES_REQUESTED** [review]
Some code clean up suggestions

**Comment by pbolduc** [comment]
Also noticed there are not tests associated with the front end code.

File: src/frontend/staff-portal/debug.log (hunk 1 above)
Comment (lfdgov) [reply src/frontend/staff-portal/debug.log]: done

File: src/frontend/staff-portal/src/app/services/lookups.service.ts (hunk 3 above)
Comment (lfdgov) [reply src/frontend/staff-portal/src/app/services/lookups.service.ts]: done

**Comment by lfdgov** [comment]
>

**Review by pbolduc — APPROVED** [review]
thanks for making the changes

**Pull request merged**